src/components/SeasonStandings.vue

```diff
@@ -248,6 +249,47 @@ export default {
       ],
     };
   },
+  methods: {
+    calcTotalInPlayers() {
+      this.players.map((el) => {
+        el.total = 0
+        for (let key in el) {
+          if (key.substring(0,4) === 'game' || key === 'bye') {
```

Comment:
Not sure if this will work for all the cases. With this code, if the property starts with `game` or is equal to `bye`, it'll add to the total. I mean, I tested it and it calculates the correct total, but is that enough?
Please confirm it @reddock